app/models/user.rb

```diff
@@ -1,6 +1,7 @@
 # Represents a user in the application. Users are shared across all instances.
 class User < ActiveRecord::Base
   include UserAuthenticationConcern
+  include UserOmniauthConcern
```

Comment:
The method overriding(`#new_with_session`) is not working after I include this in the authentication concern.

I'm lean towards to just merge two concerns.